### PR TITLE
Add PC Building Simulator

### PR DESCRIPTION
**Pull request by Zemogiter**
The only issue with this game is the benchmark animations are not displayed. The benchmark itself works. Tried media foundation, DXVK, other wine versions.

### CODE REVIEW

**Comment by ImperatorS79** [comment]
DXVK will not help since it is a d3d9 game.

File: Applications/Games/PC Building Simulator/Steam/script.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+include("engines.wine.quick_script.steam_script");
+include("engines.wine.plugins.virtual_desktop");
+include("engines.wine.verbs.corefonts");
+include("utils.functions.apps.resources");
+
+var installerImplementation = {
+    run: function () {
+        new SteamScript()
+            .name("PC Building Simulator")
+            .editor(" Claudiu Kiss, The Irregular Corporation")
```
Comment (plata) [inline Applications/Games/PC Building Simulator/Steam/script.js]: Remove leading space.

File: Applications/Games/PC Building Simulator/Steam/script.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            .author("Zemogiter")
+            .applicationHomepage("http://www.pcbuildingsim.com/")
+            .wineDistribution("upstream")
```
Comment (plata) [inline Applications/Games/PC Building Simulator/Steam/script.js]: upstream is default

**Comment by plata** [comment]
@ImperatorS79 anything you'd like to check? Otherwise I think we can merge this if the app works.

**Comment by Zemogiter** [comment]
I want to test if installing DXVK or VK9 helps with the microstutter.

**Comment by ImperatorS79** [comment]
VK9 is too alpha and DXVK is not for d3d9 ^^.

**Pull request merged**